milestone-delivery/deliveries/milestone-delivery/deliveries/miden-5.md

```diff
@@ -0,0 +1,14 @@
+# Milestone Delivery :mailbox:
+
+> Only the GitHub account, which is responsible for the pull request of the accepted application is allowed to submit milestones. Don't remove any of the mandatory parts presented in bold letters or as headlines!
+
+**The Ethereum address has been filled out correctly for this milestone and the delivery is according to the official [milestone delivery guidelines](../).**  
+
+* **PR Link:** https://github.com/vechain/grant-program/pull/135
+* **Milestone Number:** 5
```

Comment:
Could you updating the milestone number accordingly? It should be Milestone 2 after the adjustment.

Reply:
@libotony updated as requested. See commit 5c70dae217c0d8a0d5ebb5c13f18f90eb7622a62